src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/GroupModelRunParametersController.kt

```diff
@@ -43,4 +48,31 @@ class GroupModelRunParametersController(
 
         return objectCreation(context, "/modelling-groups/$groupId/model-run-parameters/$id/")
     }
+
+    fun getModelRunParameterSet(): StreamSerializable<ModelRun>
+    {
+        val setId = context.params(":model-run-parameter-set-id")
+        val data = getModelRunParametersAndMetadata()
+        val filename = "set_$setId.csv"
+        context.addAttachmentHeader(filename)
+        return data
+    }
+
+
+    private fun getModelRunParametersAndMetadata():
```

Comment:
This all looks perfect, just as discussed switch to getting the touchstone using the touchstone repo rather than the modelling group repo, also this function isn't getting any meta data, so should rename to just `getModelRunParametersDatatable`